### DISCUSSION TITLE
Will Nuxt Content v3 support localization or `nuxt-i18n` compatibility?

### DISCUSSION THREAD

**Discussion by larrasu** [nuxt/content#2861] [Q&A] [answered]
In Nuxt Content v2, implementing localization was challenging as it lacked clear compatibility with `nuxt-i18n`, and there were no official best practices or examples for integrating the two effectively.

Will Nuxt Content v3 address these issues by providing support for localization or clearer guidance for integrating it with `nuxt-i18n`?

Any insights or updates on this would be greatly appreciated.

**Comment by narr07** [comment]
@farnabaz how about this?

**Comment by ptdev** [comment]
Hey,

Just adding my 2 cents here. This would be awesome to eventually support.

Ideally, integrating both modules to somehow automate/simplify what's done by nuxt-i18n's [defineI18nRoute](https://i18n.nuxtjs.org/docs/guide/custom-paths#page-component) and [setI18nParams](https://i18n.nuxtjs.org/docs/guide/custom-paths#dynamic-route-parameters) using front-matter parameters to generate the routing would be great if possible.

Cheers and thanks for all the work on this module ❤️

**Comment by narr07** [comment, marked as answer]
Nuxt Content support is finally here! Check out this example @larrasu 

[https://github.com/nuxt/content/tree/v3/examples%2Fi18n](https://github.com/nuxt/content/tree/v3/examples%2Fi18n)
